src/async/deserializeAsync.ts

```diff
@@ -239,8 +189,8 @@ export function createTsonParseAsyncInner(opts: TsonAsyncOptions) {
 					);
 
 					// cancel all pending promises
-					for (const deferred of cache.values()) {
-						deferred.next.reject(err);
+					for (const controller of cache.values()) {
+						controller.error(err);
```

Comment:
Can you call `.error()` after it's closed?

Reply:
`.error` yes I believe, this method just makes it so that any future interaction with the stream will error, it is not an error by itself.